content/en/core/releases/3.17.0.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "3.17.0 release notes"
+linkTitle: "3.17.0"
+weight:
+description: >
+relevantLinks: >
+toc_hide: true
+---
+
+## Known issues
+
+Check the repository for the [latest known issues](https://github.com/medic/cht-core/issues?q=is%3Aissue+label%3A%22Affects%3A+3.17.0%22).
+
+## Upgrade notes
+
+### Breaking changes
+
+None.
+
+### UI/UX changes
+
+#### Labels of action buttons are now available in small screens
+
+A number of commonly used phones have a horizontal resolution of 720 with a default dp of 360. We have enabled the labels of action buttons for those small screens to make actions easier to understand.
+
+![](../images/3.17.0-action-buttons.png)
+
+- [#7721](https://github.com/medic/cht-core/issues/7721): Icon labels on action buttons are not showing at the default dp for commonly used mobile devices.
+
+#### Report's filter redesign
+
+The report's filter redesign is the first step of incremental UI/UX updates to make CHT Framework more scalable, intuitive and mobile-first. The old design is still available by enabling the [permission]({{< ref "apps/reference/app-settings/user-permissions" >}}) `can_view_old_reports_filter` to the users, however the old design is considered deprecated and will be _completely removed soon_ in a future release. 
+
+![](../images/3.17.0-reports-filter-redesign.png)
+
+- [#7653](https://github.com/medic/cht-core/issues/7653): Report's filter redesign.
+
+## Highlights
+
+### Places created with bulk upload now have a primary contact
+
+We have improved the [bulk upload]({{< ref "apps/guides/data/users-bulk-load" >}}) feature to automatically assign a primary contact to a newly created place.
+
+- [#7724](https://github.com/medic/cht-core/issues/7724): Places created with bulk upload don't have a primary contact.
+
+### Privacy policy is now public
+
+Android app submission now requires a URL to the apps' privacy policy. To facilitate this process we have made the [privacy policy]({{< ref "apps/guides/privacy/privacy-policy" >}}) public in the login page.
+
+![](../images/3.17.0-privacy-policy.png)
+
+- [#7662](https://github.com/medic/cht-core/issues/7662): Expose privacy policy at a public URL
+
+## And more...
```

Comment:
This section here is looking a bit empty 😅 but it's according to pre-release notes format

Reply:
There's something weird here because some issues are listed twice, and some only once. Is it because UI/UX changes are pulled out? If so I think it might be a bug in the script. Generally we list _everything_ in the "and more..." section, even if they're also listed in the upper sections.

Could you please check and fix the script if necessary? And consider renaming the "and more..." section to "full list" (or similar?

Reply:
I tried the follow previous releases structure for UI/UX, I think it was 3.14 or 3.15... I'm going to list the UI/UX issues in the "Full list" section

Reply:
"Full list" sounds okay to me, it's simple. I tried "Complete list", "Full list of issues" but sounds too much to me.